broker/cloud_functions/lvk/store_BigQuery/deploy.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+# Deploys or deletes broker Cloud Function
+# This script will not delete Cloud Functions that are in production
+
+testid="${1:-test}"
+# "False" uses production resources
+# any other string will be appended to the names of all resources
+teardown="${2:-False}"
+# "True" tearsdown/deletes resources, else setup
+survey="${3:-lvk}"
+# name of the survey this broker instance will ingest
+versiontag="${4:-O4}"
+
+#--- GCP resources used in this script
+store_bq_trigger_topic="${survey}-alerts"
+store_bq_CF_name="${survey}-store_in_BigQuery"
```

Comment:
Let's use lower case for all resource names.

```suggestion
store_bq_CF_name="${survey}-store_in_bigquery"
```